spec/system/planning_applications/assessing/planning_history_show_spec.rb

```diff
@@ -14,111 +14,57 @@
     let!(:planning_application) do
       create(:planning_application, :in_assessment, uprn: "100081043511", local_authority: default_local_authority)
     end
+    let!(:site_history) { create(:site_history, planning_application:) }
+    let!(:refused_site_history) { create(:site_history, :refused, planning_application:) }
```

Comment:
Tiny nitpick: these can be `let` instead of `let!` (created on-demand instead of in advance).

```suggestion
    let(:site_history) { create(:site_history, planning_application:) }
    let(:refused_site_history) { create(:site_history, :refused, planning_application:) }
```